resources/ext/config/request-logging.xml

```diff
@@ -0,0 +1,9 @@
+<configuration debug="false">
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <file>/var/log/puppetserver/puppetserver-access.log</file>
```

Comment:
Jeremy's suggestion is correct

(https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppetserver)